ingenious-pages/app/Resources/views/catalog.html.twig

```diff
@@ -5,7 +5,7 @@
                         <h3 class="page-title">My Albums</h3>
                         <div class="cat-album-container">
                             {% for album in catalog|reverse %}
-                            {% if album.id != profilePicturesID and album.id != coverPicturesID and album.state == 'active' %}
+                            {% if album.id != profilePicturesID and album.id != coverPicturesID and album.id != articlePicturesID and album.state == 'active' %}
```

Comment:
In one of the meetings, we discussed moving this logic out of the page and in to the query that gets the album.

I've created MRADEV-795 and put it in the backlog for future work.